bokeh/settings.py

```diff
@@ -326,6 +348,12 @@ def nodejs_path(self, default=None):
 #:   to an affirmative value will result in more friendly simple numeric IDs
 #:   counting up from 1000.
 #:
+#: ``BOKEH_USE_BINARY_ARRAYS`` --- Whether to encode data arrays before sending them.
+#:
+#:   Accepted values are ``yes``/``no``, ``true``/``false`` or ``0``/``1``.
+#:   Can have a huge effect on performance because encoding and decoding
+#:   arrays as raw strings can be very slow.
```

Comment:
There's no mention of any reason to turn this off, here; I think if we can't say why you'd turn it off, it shouldn't be configurable. Being able to toggle this means eventually the non-default option will be broken in a release, probably, because nobody tested it...